e2e/engine/cmd.go

```diff
@@ -337,6 +340,43 @@ func checkConditions(e *engineCmd, n lib.TestCase, out []byte) error {
 				return fmt.Errorf("❌ expect output to contain '%s' got '%s'", expected, string(out))
 			}
 		}
+
+		if n.Json {
+			var jsonKeys []string
+			var results []string
+			for i, expectedJson := range n.ExpectedJson {
+				t, err := template.New("expectedjson").Parse(expectedJson)
+				if err != nil {
+					return err
+				}
+				buf := new(bytes.Buffer)
+				err = t.Execute(buf, e.conf)
+				if err != nil {
+					return err
+				}
+				if i%2 == 0 {
+					jsonKeys = append(jsonKeys, buf.String())
+				} else if i%2 == 1 {
```

Comment:
Seeing as there are only two possibilities (either `i%2 == 0` or it's `1`) this should just be an `else`